lib/routes/routeBackbeat.js

```diff
@@ -1657,5 +1686,19 @@ function routeBackbeat(clientIP, request, response, log) {
         ));
 }
 
+function getAuthLogFields(userInfo) {
```

Comment:
There's another one with unification

```js
                if (isRequesterASessionUser(userInfo)) {
                    authNames.sessionName = userInfo.getShortid().split(':')[1];
                }
```

But I think the session name is already present in userName